recipes-core/images/iot2050-image-qemu.bb

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Enes Çolpan <enes.colpan@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require recipes-core/images/iot2050-image-example.bb
+require recipes-core/images/swupdate.inc
+
+inherit image_uuid
+
+IMAGE_TYPEDEP:wic += "squashfs"
+
+WKS_FILE = "iot2050-qemu.wks.in"
+
+IMAGE_FSTYPES += "swu"
+
+WIC_IMAGER_INSTALL += "efibootguard"
+# watchdog is managed by U-Boot - disable
+WDOG_TIMEOUT = "0"
+WICVARS += "WDOG_TIMEOUT KERNEL_IMAGE INITRD_DEPLOY_FILE DTB_FILES"
+
+INITRD_DEPLOY_FILE = "${INITRAMFS_RECIPE}-${DISTRO}-${MACHINE}.initrd.img"
+
+# not compatible with SWUpdate images
+IMAGE_INSTALL:remove = "regen-rootfs-uuid"
+IMAGE_INSTALL:remove = "install-on-emmc"
+
+# EFI Boot Guard is used instead
+IMAGE_INSTALL:remove = "u-boot-script"
+
+IMAGE_INSTALL += "efibootguard"
+IMAGE_INSTALL += "swupdate"
+IMAGE_INSTALL += "swupdate-handler-roundrobin"
+IMAGE_INSTALL += "swupdate-complete-update-helper"
+IMAGE_INSTALL += "iot2050-watchdog"
+
+DTB_FILES = ""
```

Comment:
This is really too much duplication and will be hard to maintain, constantly broken. Focus on the real differences.